tsconfig.json

```diff
@@ -1,7 +1,7 @@
 {
   "compilerOptions": {
-    "target": "ES2021",
-    "lib": ["ES2021"],
+    "target": "ES2020",
+    "lib": ["ES2020"],
     "moduleResolution": "nodenext",
     "module": "nodenext",
```

Comment:
Arguably this should be set to `node16` since the "*next" configurations are typically moving targets based on the TS version. I successfully use `node16` module mode on Node 14 (the setting was renamed from `node12` before the last release).

Reply:
We need mjs support from TS.

Reply:
Yes, both nodenext and node16 do that. For example, the project where I'm forced to use mjs because zx doesn't support CommonJS: https://github.com/jakebailey/typeformer/blob/master/tsconfig.json